leapp/libraries/stdlib/__init__.py

```diff
@@ -30,3 +40,31 @@ def call(args, split=True):
     if split:
         return r.splitlines()
     return r
+
+
+def _logging_handler(fd, buffer):
+    if os.getenv('LEAPP_DEBUG', '0') == '1':
+        if fd == call.STDOUT:
+            sys.stdout.write(buffer)
+        else:
+            sys.stderr.write(buffer)
+
+
+def new_call(args, split=False):
+    _id = str(uuid.uuid4())
```

Comment:
I'm wondering why are we creating the ID here, and not in the database.

Reply:
Because it is not a ID in sense of a key in the db but a correlation ID for the call to be able to correlate multiple events.